src/Blocks/Expandable.purs

```diff
@@ -2,16 +2,39 @@ module Ocelot.Block.Expandable where
 
 import Prelude
 
-import DOM.HTML.Indexed (HTMLheader, HTMLspan, HTMLdiv)
+import DOM.HTML.Indexed (HTMLdiv, HTMLheader, HTMLspan, Interactive)
+import Data.Array (snoc)
+import Data.Bifunctor (lmap, rmap)
+import Data.Foldable (foldr)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.Tuple (Tuple(..))
+import Halogen.HTML (PropName(..))
 import Halogen.HTML as HH
+import Halogen.HTML.Core (class IsProp, Prop(..), PropValue)
 import Halogen.HTML.Properties as HP
+import Halogen.VDom.DOM.Prop (propFromString)
 import Ocelot.Block.Icon as Icon
 import Ocelot.HTML.Properties ((<&>))
+import Unsafe.Coerce (unsafeCoerce)
 
 data Status
   = Collapsed
   | Expanded
 
+renderStatus :: Status -> String
```

Comment:
`render*` has an overloaded meaning with regards to Halogen and components. Without looking at the implementation / types I'd expect this to be a render function. 

Worth renaming to `statusToString` or `showStatus`?

We could also put the data type in its own module and then expose a `toString` function there. You're expected to import qualified, so `Status.toString`.

Should we have a convention for what these functions are named? We have lots of them all over the place.